docs/docusaurus/docs/core/expectation_suites/manage_expectation_suites.md

```diff
@@ -0,0 +1,294 @@
+```
+
+</p>
+</details>
+
+## Get an Expectation
+
+1. Import the GX Core library and `expectations` module.
+
+  ```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py imports"
+  ```
+
+2. Get a Data Context.
+
+  ```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py get_context"
+  ```
+
+3. [Get an existing Expectation Suite](#get-an-existing-expectation-suite) that contains Expectations, or [create a new Expectation Suite](#create-a-new-expectation-suite) and [add some Expectations to it](#add-expectations-to-an-expectation-suite).  This example retrieves an existing Expectation Suite.
+  ```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py retrieve Expectation Suite"
+  ```
+
+4. Find the desired Expectation by iterating the Expectation Suite's Expectations and comparing classes and attributes to those of the desired Expectation.
+  ```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py retrieve expectation"
+  ```
+
+<details><summary>Full example code</summary>
+<p>
+
+```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py full example code"
+```
+
+</p>
+</details>
+
+## Edit a single Expectation
+
+1. Import the GX Core library and the `expectations` module.
+  ```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py imports"
+  ```
+
+2. Get a Data Context
+  ```python name="core/expectation_suites/_examples/edit_a_single_expectation.py get data context"
+  ```
+
+3. [Get the Expectation to edit](#get-a-specific-expectation-from-an-expectation-suite) from its Expectation Suite.
+  ```python name="core/expectation_suites/_examples/edit_a_single_expectation.py get expectation to edit"
+  ```
+
+5. [Modify the Expectation](/core/expectations/manage_expectations.md#modify-an-expectation).
+  ```python name="core/expectation_suites/_examples/edit_a_single_expectation.py edit attribute"
+  ```
+
+6. Save the modified Expectation in the Expectation Suite.
+  ```python name="core/expectation_suites/_examples/edit_a_single_expectation.py save the Expectation"
+  ```
+  :::info
+  `expectation.save()` is explicitly used to update the configuration of an Expectation in an Expectation Suite.  If the Expectation is not part of an Expectation Suite, `expectation.save()` will fail.
+  
+  You can [test changes to an Expectation](/core/expectations/manage_expectations.md#test-an-expectation) without running `expectation.save()`, but those changes will not persist in the Expectation Suite until `expectation.save()` is run.
+  :::
+
+<details><summary>Full example code</summary>
+<p>
+
+```python name="core/expectation_suites/_examples/edit_a_single_expectation.py full example code"
+```
+
+</p>
+</details>
+
+## Edit multiple Expectations
```

Comment:
```suggestion
## Edit multiple Expectations
```
Remove. This is covered in Manage Expectations.

Reply:
I think that this should remain under manage Expectation Suites because the workflows they describe require an Expectation Suite -- eg, Edit multiple Expectations _in an Expectation Suite._. This is a "thing you can do with an Expectation Suite" rather than strictly a "Thing you can do with an Expectation".